tests/common/test_core.py

```diff
@@ -1,13 +1,17 @@
+import pytest
+
 import doctr
 
 
 def test_version():
     assert len(doctr.__version__.split(".")) == 3
 
 
+@pytest.mark.skipif(doctr.is_torch_available() and doctr.is_tf_available(), reason="torch and tf are available")
```

Comment:
@odulcy-mindee this is better it skips the tests (`make tests` works now also if both backends installed if you develop on doctr) and does not effect the CI jobs

Reply:
Oh yeah, that's even better